src/Transcriber.js

```diff
@@ -12,14 +12,15 @@ class Transcriber extends EventEmitter {
     audioContext = window.AudioContext || window.webkitAudioContext,
     microphone = Microphone,
     maxFinalWait = 5000,
-    audioParams = null,
-    relayParams = null,
+    audioParams = {},
+    relayParams = {},
   } = {}) {
     super();
     this.gkUrl = gkUrl;
-    this.getUserMedia = getUserMedia || (navigator.mediaDevices && navigator.mediaDevices.getUserMedia.bind(
-      navigator.mediaDevices
-    ));
+    this.getUserMedia =
+      getUserMedia ||
+      (navigator.mediaDevices &&
+        navigator.mediaDevices.getUserMedia.bind(navigator.mediaDevices));
```

Comment:
guessing we didn't run prettier on these previously 😄

Reply:
Yeah, I've got it integrated with my editor now.